src/Cosmonaut/CosmosStoreSettings.cs

```diff
@@ -58,13 +60,15 @@ public class CosmosStoreSettings
             string authKey,
             ConnectionPolicy connectionPolicy = null,
             IndexingPolicy indexingPolicy = null,
-            int defaultCollectionThroughput = CosmosConstants.MinimumCosmosThroughput)
+            int defaultCollectionThroughput = CosmosConstants.MinimumCosmosThroughput,
+            UniqueKeyPolicy uniqueKeyPolicy = null)
```

Comment:
Can you remove that from the ctor? I am moving away from optional parameters via the ctor

Reply:
No problem. I just followed the patterns that were already there.
I’ll remove it tomorrow.